com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/SpriteForwardPass.hlsl

```diff
@@ -21,6 +25,21 @@ half4 frag(PackedVaryings packedInput) : SV_TARGET
     half4 color = half4(surfaceDescription.BaseColor, surfaceDescription.Alpha);
 #endif
 
+    #if defined(DEBUG_DISPLAY)
+    SurfaceData2D surfaceData;
+    InitializeSurfaceData(color.rgb, color.a, surfaceData);
+    InputData2D inputData;
+    InitializeInputData(unpacked.positionWS, unpacked.texCoord0, inputData);
+    half4 debugColor = 0;
+
+    SETUP_DEBUG_DATA_2D(inputData, unpacked.positionWS);
+
+    if(CanDebugOverrideOutputColor(surfaceData, inputData, debugColor))
```

Comment:
```suggestion
    if (CanDebugOverrideOutputColor(surfaceData, inputData, debugColor))
```

Reply:
Thanks, fixed a bunch of these in one go.